opendistro-for-elasticsearch/anomaly-detection
# fix failed integration cases

*Issue #, if available:*

## Description of changes:
Fix flaky integration test cases

## Test
1. Start local cluster with security plugin and run `./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=true -Dsecurity=true -Duser=admin -Dpassword=admin`
2. Start local cluster without security plugin and run `./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=false` and `./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster"`
3. `./gradlew build`
4. `./gradlew integTest -PnumNodes=3`
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
